features/bookmarks/bookmark_browse.feature

```diff
@@ -16,3 +15,11 @@ Feature: Browse Bookmarks
     When I go to the bookmarks page for user "ethel" with pseud "aka"
     Then I should see "Bookmarked with Other Pseud"
       And I should not see "Bookmarked with Default Pseud"
+
+  Scenario: When logged out, the "save" button on bookmarks does not show
+    Given I am logged in as "bookmarker"
+      And I bookmark the work "Test" with the tags "testing"
```

Comment:
This test doesn't fail when the change is reverted. This is an indication that it's not testing the right thing.

In particular, this creates a bookmark with the bookmarker tag "testing". To reproduce the issue, a bookmark in the testing fandom is needed. For that, the fandom needs to be canonical, there needs to be a work in the fandom and then that work should be bookmarked.